pkgs/development/python-modules/keystone-engine/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, stdenv, buildPythonPackage, fetchPypi, keystone }:
+
+buildPythonPackage rec {
+  pname = "keystone-engine";
+  version = "0.9.2";
+
+  src = fetchPypi {
+   inherit pname version;
+   sha256 = "1xahdr6bh3dw5swrc2r8kqa8ljhqlb7k2kxv5mrw5rhcmcnzcyig";
+  };
+
+  preConfigure = ''
+    sed -i "s;libkeystone.so;${keystone}/lib/libkeystone.so;" setup.py
```

```suggestion
    substituteInPlace setup.py --replace \
       "libkeystone.so;" "${keystone}/lib/libkeystone.so;" 
```

This has the benefit that we get a warning, when the substitution does not apply anymore.

---

Fixed.